iCubGenova04/icub_wbd_inertials.xml

```diff
@@ -76,6 +76,19 @@
     <!-- WRAPPERS FOR INERTIAL SENSORS ON MTB, EMS AND FT STRAIN BOARDS -->
     <xi:include href="wrappers/inertials/left_leg-inertials_wrapper.xml" />
 
+    <!-- DEVICES FOR INERTIAL SENSORS ON MTB, EMS AND FT STRAIN BOARDS -->
+    <devices file="hardware/inertials/right_leg-eb8-IMU.xml" />
+    <devices file="hardware/inertials/right_leg-eb8-inertials.xml" />
+    <devices file="hardware/inertials/right_leg-eb9-IMU.xml" />
+    <devices file="hardware/inertials/right_leg-eb9-inertials.xml" />
+    <devices file="hardware/inertials/right_leg-eb11-inertials.xml" />
```

Comment:
Are these being done for the left leg as well?

Reply:
The corresponding devices for the left leg already seem to exist in https://github.com/robotology/robots-configuration/blob/910d6b2785d7ad7556d900c86aec3a2f982d3025/iCubGenova04/icub_wbd_inertials.xml#L66. Thank you!